### DISCUSSION TITLE
where are methods in NeighborList.cpp called/invoked

### DISCUSSION THREAD

**Discussion by eric-h-kang** [deepmodeling/deepmd-kit#122] [Q&A] [answered]
I am trying to modify the source of the DPMD code to experiment with more limited data sets, but I cannot see where the methods in /source/lib/src/NeighborList.cpp are called. Any insight would be greatly appreciated

**Comment by njzjz** [comment, marked as answer]
https://github.com/deepmodeling/deepmd-kit/blob/6b9c29ecfbdcd84908a0673dc7906d7d6653c32b/source/op/descrpt_se_a.cc#L263
Do you mean this line?
